relay/relay.go

```diff
@@ -387,6 +390,11 @@ func (r *Relay) handleEvents(ctx context.Context, reader2 reader.Reader, transfo
 					cfg := r.cfg.From
 					r.tctx.L().Error("the requested binlog files have purged in the master server or the master server have switched, currently DM do no support to handle this error",
 						zap.String("db host", cfg.Host), zap.Int("db port", cfg.Port), log.ShortError(err))
+					// log the status for debug
+					pos, gs, err2 := utils.GetMasterStatus(r.db, r.cfg.Flavor)
+					if err2 == nil {
+						r.tctx.L().Info("current master status", zap.Stringer("position", pos), zap.Stringer("GTID sets", gs))
```

Comment:
```suggestion
						r.tctx.L().Info("current master status", zap.Stringer("position", pos), log.WrapStringerField("GTID sets", gs))
```

Reply:
done in https://github.com/pingcap/dm/pull/335/commits/9611a0c1c71645fe6b77aaaadb1ea19c1136e7b7.